pscheduler-server/pscheduler-server/api-server/pschedulerapiserver/admin.py

```diff
@@ -111,3 +113,57 @@ def mtu_safe():
     except Exception as ex:
         log.exception()
         return error(str(ex))
+
+@application.route("/status", methods=['GET'])
+def get_status():
+    response = {}
+
+    response["time"] = str(pscheduler.datetime_as_iso8601(datetime.datetime.now()))
+
+    # query process table
+    services = {}
+    items = ["scheduler", "archiver", "ticker", "runner", "database"]
+    for proc in psutil.process_iter():
+	pinfo = proc.as_dict(attrs=['name', 'create_time'])
+        if (pinfo["name"] in items):
+            # calculate elapsed running time
+            running_time = pscheduler.seconds_as_timedelta(time.time() - pinfo["create_time"])
+            services[pinfo["name"]] = { "running": True, "time": str(pscheduler.timedelta_as_iso8601(running_time)) }
+            items.remove(pinfo["name"])
+    try:
+        # query database, calculate server run time
+        cursor = dbcursor_query("SELECT extract(epoch from current_timestamp - pg_postmaster_start_time())", onerow=True)
```

Comment:
Check the indentation here for tabs vs. spaces.   See [PEP-8](https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces).

Reply:
Fixed in 0b29f07 and 1428431.